### PR TITLE
doc: reference in SuppressionXpathFilter doc about ability to generate suppressions by CLI

### PR DESCRIPTION
**Pull request by timurt**
https://github.com/checkstyle/checkstyle/issues/5980

### CODE REVIEW

File: src/xdocs/config_filters.xml [hunk 1]
```diff
@@ -737,6 +737,13 @@ public class UserService {
           Note, that support for other Checks will be available after resolving
           <a href="https://github.com/checkstyle/checkstyle/issues/4830">issue 4830</a>.
         </p>
+        <p>
+          You can use the command line helper tool to generate xpath
+          suppressions based on your configuration file and input class file.
+          See
+          <a href="http://checkstyle.sourceforge.net/cmdline.html">here</a>
```
Comment (rnveach) [inline src/xdocs/config_filters.xml]: This doesn't need to be full path. `cmdline.html` should be fine.

File: src/xdocs/config_filters.xml [hunk 2]
```diff
@@ -737,6 +737,13 @@ public class UserService {
           Note, that support for other Checks will be available after resolving
           <a href="https://github.com/checkstyle/checkstyle/issues/4830">issue 4830</a>.
         </p>
+        <p>
+          You can use the command line helper tool to generate xpath
+          suppressions based on your configuration file and input class file.
```
Comment (rnveach) [inline src/xdocs/config_filters.xml]: `input class file` => `input files`.

File: src/xdocs/config_filters.xml (hunk 1 above)
Comment (timurt) [reply src/xdocs/config_filters.xml]: done

File: src/xdocs/config_filters.xml (hunk 2 above)
Comment (timurt) [reply src/xdocs/config_filters.xml]: done

**Pull request merged**